Makefile

```diff
@@ -37,4 +43,4 @@ container-codegen:
 
 clean:
 	@rm -rf ./bin
-
+	@rm -f ./pkg/sqlite/migrations.go
```

Comment:
like the rest of our codegen, we probably want to go ahead and commit these artifacts.